server/server.go

```diff
@@ -249,7 +258,59 @@ func runServer(ctx context.Context, args []string, dEnv *env.DoltEnv) (*svcs.Con
 
 	sqlserver.ConfigureServices(serverConfig, controller, Version, dEnv)
 	go controller.Start(newCtx)
-	return controller, controller.WaitForStart()
+	
+	err = controller.WaitForStart()
+	if err != nil {
+		return nil, err
+	}
+	
+	// TODO: shutdown replication cleanly when we stop the server
+	_, err = startReplication(serverConfig)
```

Comment:
_Super small nit_ but maybe name this `optionallyStartReplication`? Just calling `startReplication` makes it seem like it will _always_ start replication. Not a strong suggestion, just a preferential one.

Reply:
I'm gonna go back through for another organizational pass and will revisit this then.